packages/wallet/package.json

```diff
@@ -29,10 +29,12 @@
     "@types/base64url": "^2.0.0",
     "tsdx": "^0.13.3"
   },
+  "peerDependencies": {
+    "@transmute/universal-wallet": "^0.4.1"
```

Comment:
add to `devDependencies` as well

Reply:
* added to devDependencies
* ran `npm install` again
* ran `npm run build`
* ran `npm test`, 17/17 passing